README.md

```diff
@@ -1,2 +1,56 @@
 # sustainable-society
 Collection of documents to use when needing an advice regarding sustainability 
+
+## Food
+-  Buy local and seasonal food, no need to buy summer food in winter, or if you do have cravings for it buy (or do it yourself) pickled or in naother way preserved version
+
+- Avoid supermarkets for most of the things, buy there only the things that are not available locally
+
+- Eat less meat, it's more sustainable and healthier
+
+## Clothing
+-  Prefer clothing brands that promote unisex, one-size-fits-all models, that are against fast fashion and use quality materials that are durable, so you can wear e.g. one T-shirt for a long time, or even swap with friends or let your kids inherit it. Example [Older brother]( https://olderbrother.us) or [Patagonia](https://www.patagonia.com/home/)
+
+- Use [eco-egg](https://www.ecoegg.com) instead of detergents to wash the clothes
+
+- Develop a style and then keep smaller amount of clothing in your closet (20 pieces e.g.) instead of just hoarding something that you barely wore once
+
+- If you want to get rid of piece of clothes give it away if it's in a good shape or place it in textile recycle bin if it's busted
+
+- If a piece of clothes is suspiciously cheap investigate that company and find out why is it so cheap - they are probably misstreating their workers 
```

Comment:
ALWAYS investigate. But if it's super cheap like H&M, you can be sure that something is not right.